tests/layer_tests/tensorflow_tests/test_tf_GRUBlockCell.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import numpy as np
+
+from common.tf_layer_test_class import CommonTFLayerTest
+
+class TestTFGRUBlockCell(CommonTFLayerTest):
+    def _prepare_input(self, inputs_dict):
+        for input in inputs_dict.keys():
+            inputs_dict[input] = np.zeros(inputs_dict[input]).astype(np.float32)
+        return inputs_dict
+
+    def create_tf_gru_block_cell(self, batch_size, input_size, hidden_size):
+        #
+        #   Create Tensorflow model
+        #
+
+        import tensorflow as tf
+        tf.compat.v1.reset_default_graph()
+
+        # Create the graph and model
+        with tf.compat.v1.Session() as sess:
+            tf_x_shape = [batch_size, input_size]
+            tf_h_prev_shape = [batch_size, hidden_size]
+            tf_w_ru_shape = [input_size + hidden_size, hidden_size * 2]
+            tr_w_c_shape = [input_size + hidden_size, hidden_size]
+            tf_b_ru_shape = [hidden_size * 2]
+            tf_b_c_shape = [hidden_size]
+
+            random_generator = np.random.default_rng(2022)
+
+            x = random_generator.uniform(0, 1, tf_x_shape).astype(np.float32)
```

Comment:
It would be good to have 'x' and 'h_prev' as placeholders so you can get rid of this dummy Add and to make sure that this model is inferred by the plugin. Luckily GRUCell doesn't implement evaluate, so in this case it's not folded before the inference.